index.js

```diff
@@ -327,27 +328,30 @@ function loadSpecOutputPathContent() {
   predefinedSpec = JSON.parse(specOutputFileContent);
 }
 
+
+
 /**
  * @description Persists OpenAPI content to spec output file
  */
 function writeSpecToOutputFile() {
-  if (specOutputPath) {
-    fs.writeFileSync(specOutputPath, JSON.stringify(spec, null, 2), 'utf8');
+  if (!specOutputPath) {
+    return;
```

Comment:
not sure if this should return silently or throw an error. perhaps a `console.warn` would be an a simple yet useful addition?

I don't remember if we handle the missing `specOutputPath` somewhere else in the codebase - if we do, this code is fine.

Reply:
Thought exactly the same way. This was the current behavior when no `specOutputPath` was supplied (just changed an early return here). It just doesn't write spec to file, which we can discuss if it's correct or not.

From the readme, the specOutputPath arg: `'path/to/a/file/filename.json' - (Optional) path to a file and file name, if missing module won't write spec to the disk`. Seems to be expected, but we could change later.

Reply:
Yep, thought so since this was already working this way. For convenience, we could throw in a `console.warn` there, but it's fine if we don't.